config/initializers/session_store.rb

```diff
@@ -10,8 +10,10 @@
 servers = ApplicationConfig["REDIS_SESSIONS_URL"] || ApplicationConfig["REDIS_URL"]
 Rails.application.config.session_store :redis_store,
                                        key: ApplicationConfig["SESSION_KEY"],
+                                       domain: ApplicationConfig["APP_DOMAIN"],
```

Comment:
Will this affect the already stored keys at all?

Reply:
I tested the deployment scenarios a bunch on https://community.benhalpern.com/ and things were fine. I'm pretty sure this just flips one attribute on the cookies without otherwise affecting behavior.